components/tooltip/tooltip.vue

```diff
@@ -190,6 +190,15 @@ export default {
       default: '',
     },
 
+    /**
+     * Controls whether hover/focus causes the tooltip to appear.
+     * Cannot be combined with the show prop. show value will be ignored.
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
hmm maybe not a great name since "disabled" is a valid html attribute on a lot of elements. Maybe we make it "enabled" defaulting to true instead.

Reply:
changed.